res/xml/urdu phonetic

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="Urdu phonetic" script="urdu">
+  <row>
+    <key key0="ق" key2="1" key4="esc"/>
+    <key key0="و" key2="2" key4="redo"/>
+    <key key0="ع" key1="!" key2="3"/>
```

Comment:
The layout contains 1, 2, 3, 6 but not 4, 5, 7, 9.

Reply:
added